velox/functions/prestosql/window/tests/SimpleAggregatesTest.cpp

```diff
@@ -58,6 +58,12 @@ class AggregateWindowTestBase : public WindowTestBase {
   void testWindowFunction(
       const std::vector<RowVectorPtr>& vectors,
       const std::vector<std::string>& frameClauses) {
+    // DuckDb returns null for empty frames, which is incorrect for the count
+    // aggregate.
+    if ((function_.rfind("count", 0) == 0) &&
```

Comment:
This condition shouldn't be added at this layer.

For the test with count() functions, you might want to add a custom result verification with Presto results.

Reply:
Thanks for the suggestion, would it be fine if an e2e test is added in presto-native-execution following this PR? This would help verify the results against Presto.
Could you please share where this check could be moved to? This was added as a temporary fix instead of removing the existing unit tests for `count()`.